templates/account/payments.php

```diff
@@ -14,15 +14,11 @@
 	 * @var array $VARS
 	 * @var Freemius $fs
 	 */
-	$fs = freemius( $VARS['id'] );
+    $fs       = freemius( $VARS['id'] );
+    $payments = freemius( $VARS['payments'] );
 
 	$slug = $fs->get_slug();
 
-	$payments = $fs->_fetch_payments();
-
-	$show_payments = ( is_array( $payments ) && 0 < count( $payments ) );
```

Comment:
@vovafeldman I have removed this code from here as we already fetching the data on accounts page. 

So, passing the data from accounts page here and retrieving the data using `freemius( $VARS['payments'] );`

Reply:
@msladu good catch 👍